autokeras/hypermodel/graph.py

```diff
@@ -463,4 +476,69 @@ def hyper_build(self, hp):
         outputs = []
         for output_node in self.outputs:
             outputs.append(old_node_to_new[output_node])
-        return PlainGraph(inputs, outputs, override_hps=self.override_hps)
+        return PlainGraph(inputs=inputs,
+                          outputs=outputs,
+                          override_hps=self.override_hps)
+
+    def get_config(self):
+        config = super().get_config()
+        block_classes = [block.__class__.__name__ for block in self.blocks]
```

Comment:
Can blocks have arguments? If so, should this be `[block.get_config() for block in self.blocks]`